contracts/MyMultiSig.cdc

```diff
@@ -264,8 +264,11 @@ pub contract MyMultiSig {
             emit ActionCreated(uuid: uuid, intent: action.intent)
             return uuid
         }
-
+            
         access(account) fun addSigner(signer: Address) {
+            // Checks if the signer exists, get account key will fail otherwise
+            getAccount(signer).keys.get(keyIndex: 0)
```

Comment:
it looks like `keys.get` returns an optional value, so i think we need to have
```
assert(getAccount(signer).keys.get(keyIndex: 0) != nil: "Account does not exist").
```

Reply:
or better yet maybe a `pre` condition?

Reply:
When I test it, it returns an error
`[Error Code: 1101] cadence runtime error Execution failed:
error: get account key failed: getting account key failed: [Error Code: 1201] account not found for address 0000000000000000
getAccount(signer).keys.get(keyIndex: 0)`